uber/site_sections/accounts.py

```diff
@@ -88,7 +88,7 @@ def reset(self, session, message='', email=None):
             try:
                 account = session.get_account_by_email(email)
             except NoResultFound:
-                message = 'No account exists for email address {!r}'.format(email)
+                message = 'The ID/Password combination provided is not valid {!r}'.format(email)
```

Comment:
This particular change is insufficient, because on the ``reset`` page they ONLY enter an email address.  So if we want people to not know what email addresses exist, then instead of returning this message (which tells them conclusively that the email address does not exist), we should instead just do a

```python
raise HTTPRedirect('login?message={}', 'Your new password has been emailed to you')
```

which is what we do down below if the reset is successful.  If we give a different response for a valid vs an invalid email address then there's no point.

If we want to make it slightly less deceptive, then we can change the message in both places to read something like "A reset password has been mailed to that address, if that address exists in our system."  That way we're not outright fooling people who think they have accounts into believing they've been emailed a reset when in fact they have not.